lib/shopify_app/controller_concerns/login_protection.rb

```diff
@@ -100,7 +100,7 @@ def login_again_if_different_user_or_shop
     end
 
     def signal_access_token_required
-      response.set_header(ACCESS_TOKEN_REQUIRED_HEADER, true)
```

Comment:
I'd never heard of this convention for booleans before, perhaps we could use a constant to help explain it, e.g. `STRUCTURED_FIELD_TRUE = "?1"` ?

Reply:
Good catch. I looked into this in more detail and found that in some Shopify repos (e.g., `Shopify/app-bridge` in `user-authorized-fetch.ts`) we rely on the `X-Shopify-API-Request-Failure-Unauthorized` header having a string value of `"true"`, not the convention from the HTTP working group spec of `"?1"` and `"?0"`. So I think it's best to simply use a string value of "true" here.